modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItMetadataTest.java

```diff
@@ -47,52 +47,51 @@ protected int nodes() {
     }
 
     @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-19106 Column namings are partially broken after upgrading to calcite 1.34")
     public void trimColumnNames() {
-        String var300 = generate(() -> "X").limit(300).collect(joining());
-        String var256 = "'" + var300.substring(0, 255);
+        String var300 = "'" + generate(() -> "X").limit(300).collect(joining()) + "'";
+        String var256 = var300.substring(0, Math.min(var300.length(), MAX_LENGTH_OF_ALIASES));
 
-        assertQuery("select '" + var300 + "' from person").columnNames(var256).check();
+        assertQuery("select " + var300 + " from person").columnNames(var256).check();
     }
 
     @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-19106 Column namings are partially broken after upgrading to calcite 1.34")
     public void columnNames() {
         assertQuery("select (select count(*) from person), (select avg(salary) from person) from person")
                 .columnNames("EXPR$0", "EXPR$1").check();
         assertQuery("select (select count(*) from person) as subquery from person")
                 .columnNames("SUBQUERY").check();
 
         assertQuery("select salary*2, salary/2, salary+2, salary-2, mod(salary, 2)  from person")
-                .columnNames("SALARY * 2", "SALARY / 2", "SALARY + 2", "SALARY - 2", "MOD(SALARY, 2)").check();
-        assertQuery("select salary*2 as first, salary/2 as secOND from person").columnNames("FIRST", "SECOND").check();
+                .columnNames("PERSON.SALARY * 2", "PERSON.SALARY / 2", "PERSON.SALARY + 2", "PERSON.SALARY - 2",
+                        "MOD(CAST(PERSON.SALARY AS DECIMAL(30, 15)), 2)").check();
+        assertQuery("select salary*2 as first, salary/2 as laST from person").columnNames("FIRST", "LAST").check();
```

Comment:
can we have aditionla tst to be sure that we keep case for quoted names like a `SELECT salary*2 as "SaLaRy ....."`